src/pages/studyView/charts/barChart/BarChart.tsx

```diff
@@ -140,15 +141,21 @@ export default class BarChart extends React.Component<IBarChartProps, {}> implem
     @computed
     get bottomPadding(): number {
         const MAX_PADDING = 30;
-        const padding = this.tilted ? _.max(this.tickFormat.map((tick: string | string[]) => {
+        
+        const padding = _.max(this.tickFormat.map((tick: string | string[]) => { 
             let content;
             if (_.isArray(tick)) {
                 content = tick.join();
             } else {
                 content = tick;
             }
-            return getTextWidth(content, VICTORY_THEME.axis.style.tickLabels.fontFamily, `${VICTORY_THEME.axis.style.tickLabels.fontSize}px`);
-        })) * Math.sin(Math.PI * TILT_ANGLE / 180) + 5 : 20;
+            const fontFamily = VICTORY_THEME.axis.style.tickLabels.fontFamily;
+            const fontSize = `${VICTORY_THEME.axis.style.tickLabels.fontSize}px`;
+            const textWidth = getTextWidth(content, fontFamily, fontSize);
+            const textHeight = measureText({content, fontFamily, fontSize, lineHeight: 1}).height.value;
```

Comment:
Let's have a function in `wrapText` to return width, height and diagonal for future reuse.